test/sql/is_null_sql_test.cpp

```diff
@@ -0,0 +1,164 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// is_null_sql_test.cpp
+//
+// Identification: test/sql/is_null_sql_test.cpp
+//
+// Copyright (c) 2015-17, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#include <memory>
+
+#include "catalog/catalog.h"
+#include "common/harness.h"
+#include "concurrency/testing_transaction_util.h"
+#include "executor/create_executor.h"
+#include "optimizer/optimizer.h"
+#include "planner/create_plan.h"
+#include "sql/testing_sql_util.h"
+
+namespace peloton {
+namespace test {
+
+class IsNullSqlTests : public PelotonTest {};
+
+void CreateAndLoadTable() {
+  // Create a table first
+  LOG_INFO("Creating a table...");
+  LOG_INFO("Query: create table a(id int, value int)");
+
+  TestingSQLUtil::ExecuteSQLQuery("create table a(id int, value int);");
+
+  LOG_INFO("Table created!");
+
+  // Insert multiple tuples into table
+  LOG_INFO("Inserting a tuple...");
+  LOG_INFO("Query: insert into a values(1, 1)");
```

Comment:
Please use LOG_TRACE for the logging here. (ExecuteSQLQuery function has INFO level logging for the query content and result.)